app/src/componentLibrary/sidebar/navbar/navbar.jsx

```diff
@@ -80,6 +93,11 @@ Navbar.propTypes = {
   mainControlBlock: PropTypes.element,
   footerControlBlock: PropTypes.element,
   setIsOpenPopover: PropTypes.func,
+  getClassName: PropTypes.object.isRequired,
+  onEnterButton: PropTypes.func.isRequired,
+  onLeaveButton: PropTypes.func.isRequired,
+  onButtonDown: PropTypes.func.isRequired,
+  onButtonUp: PropTypes.func.isRequired,
```

Comment:
Please align naming to be consistent with event handlers names.

Reply:
Done.